src/main/java/gregtech/api/unification/material/materials/FirstDegreeMaterials.java

```diff
@@ -1281,7 +1282,7 @@ public static void register() {
                 .build();
 
         DistilledWater = new Material.Builder(421, gregtechId("distilled_water"))
-                .fluid()
+                .liquid(new FluidBuilder().alternativeName("fluidDistWater"))
```

Comment:
Was this change from `fluid` to `liquid` intentional?

Reply:
Yes, `fluid()` is the same as calling `liquid(new FluidBuilder())`